pallets/external-validators/src/lib.rs

```diff
@@ -0,0 +1,697 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+//! ExternalValidators pallet.
+//!
+//! A pallet to manage external validators for a solochain.
+//!
+//! ## Terminology
+//!
+//! - WhitelistedValidators: Fixed validators set by root/governance. Have priority over the external validators.
+//! - ExternalValidators: Validators set using storage proofs from another blockchain. Changing them triggers a
```

Comment:
This is inaccurate right? As `set_external_validators` does not trigger anything.

Reply:
Yes, outdated, will fix

Reply:
*fixed*